x-pack/platform/plugins/shared/cases/server/connectors/cases/schema.ts

```diff
@@ -54,6 +54,14 @@ const TimeWindowSchema = schema.string({
     if (!date || !date.isValid()) {
       return 'Not a valid time window';
     }
+
+    const match = value.match(/^(\d+)([mhdw])$/);
+    const timeSize = parseInt(match[1], 10);
+    const timeUnit = match[2];
+
+    if (timeUnit === 'm' && timeSize < 5) {
+      return 'Not a valid time window';
```

Comment:
nit: Should we show similar message as UI `Time window should be at least 5 minutes` to mention minimum limit?